src/main/resources/alfresco/subsystems/Transformers/default/transformers.properties

```diff
@@ -61,10 +61,8 @@ content.transformer.complex.Any.Image.priority=400
 
 content.transformer.complex.Text.Image.pipeline=*|pdf|complex.PDF.Image
 content.transformer.complex.Text.Image.priority=350
-content.transformer.complex.Text.Image.extensions.csv.*.supported=true

```

Comment:
Source types of csv and txt were never used in Legacy. When looking at what should be supported in Local this complicated the situation.